exercise12_3.py

```diff
@@ -7,21 +7,18 @@
 Python for Everybody: Exploring Data Using Python 3
 by Charles R. Severance
 
-Solution by Jamison Lahman, June 4, 2017
+Solution by Daniel Lee, Dec 29, 2019
 """
 import urllib.request
-import urllib.parse
-import urllib.error
```

Comment:
I don't think these are necessarily needed (and I personally would prefer importing packages directly), but they were intentionally kept to match the text.